triage.md

```diff
@@ -0,0 +1,40 @@
+# Triaging of Podman issues
+To manage new GitHub issues, maintainers perform issue triage on a regular basis and categorize the issues based on priority, type of issue, and other factors.
+
+This process includes:
+1. Ensure the issue is relevant to the correct repository (i.e. build issues go to buildah repo, podman desktop issues go to Podman Desktop repo, etc) and transfer as needed.
+2. Categorize issues by type and assign its associated label (see below) and “traiged” label. If the issue is a bug and it is of high impact, please assign a high-impact label.
+3. Assign high-impact issues to either themselves or a [core maintainer](https://github.com/containers/podman/blob/main/OWNERS#L1).
+4. If essential information is lacking (see below), request it from the submitter and apply the 'needs-info' label.
```

Comment:
What should I expect to see below?  I'm not sure what you're referring to with `see below`.